hackathon_site/registration/jinja2/registration/application.html

```diff
@@ -13,6 +13,15 @@
 {% block form_title %}Application{% endblock %}
 {% block form_attrs %}enctype="multipart/form-data"{% endblock %}
 
+{% block nav_links %}
+<li><a href={{ url("event:dashboard") }}>Dashboard</a></li>
+<li><a href={{ url("event:change_password") }}>Change Password</a></li>
+{% endblock %}
+
+{% block mobile_nav_btn %}
+<a href="#" data-target="nav-mobile" class="sidenav-trigger"><i class="material-icons">menu</i></a>
```

Comment:
I'm beginning to notice that this is here a lot. Could we move it into base.html, maybe with a check for if `nav_links` is empty or not? That would be like

```jinja2
{% block mobile_nav_btn %}
{% if self.nav_links() %}
<a href="#" data-target="nav-mobile" class="sidenav-trigger"><i class="material-icons">menu</i></a>
{% endif %}
{% endblock %}
```